### ISSUE TITLE
Localization does not work

### ISSUE THREAD

**Issue by cicero200272** [nicufarmache/lovelace-big-slider-card#40]
**Checklist:**

- [X] I updated to the latest version available
- [X] I cleared the cache of my browser

**Release with the issue:**
1.1.4

**Last working release (if known):**
n/a

**Browser and Operating System:**

**Description of problem:**
Lights that are switched off appear as "off" although it should read "aus" (German locale). HA is set to Germany/German in settings.

**Javascript errors shown in the web inspector (if applicable):**

```

```

**Additional information:**

**Comment by jensrossbach** [comment]
Looks like the card uses some own localization which only supports English and Romanian. But for the standard values like 'on', 'off' etc., the card should make use of Home Assistant's localization capabilities.

**Comment by nicufarmache** [comment]
You're right. I'm going to change it to use the HA localisation in the next version

**Comment by nicufarmache** [comment]
I just found out that there is a new localisation api available from version 2023.9 https://developers.home-assistant.io/blog/2023/08/29/hass-format-state/

**Comment by nicufarmache** [comment]
I fixed this in 1.2.0 it should use the translated texts from HA now
